### PR TITLE
rename the k8s csi external snapshotter to match the other k8s csi pa…

### PR DESCRIPTION
**Pull request by patflynn**

### CODE REVIEW

**Review by imjasonh — CHANGES_REQUESTED** [review]
Let's update the subpackages too

**Comment by patflynn** [comment]
done. should we get all the other instances of k8s-csi packages into this PR as well?

**Comment by imjasonh** [comment]
Yeah anything else you think we should include

File: packages.txt [hunk 1]
```diff
@@ -677,7 +678,7 @@ kubewatch
 hey
 external-resizer
 external-attacher
-external-snapshotter
+kubernetes-csi-external-snapshotter
```
Comment (developer-guy) [inline packages.txt:681]: isn't it a duplicate entry 🤔

**Comment by patflynn** [comment]
@imjasonh good to dismiss request changsed?

**Review by imjasonh — APPROVED** [review]

**Review by developer-guy — APPROVED** [review]

**Comment by developer-guy** [comment]
eventually, we should rebuild the images as well to be able to use new package names, right? Will you do this @patflynn or I can do this for you WDYT?

**Comment by developer-guy** [comment]
I'm going to work on the kubernetes-csi-external-snapshotter images for both public (latest) and private (6.0.1), I need this one to be merged first 😁

**Comment by Dentrax** [comment]
Done! @patflynn

https://github.com/chainguard-images/images/pull/661

**Pull request merged**